sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/MapElementsTest.java

```diff
@@ -53,6 +53,29 @@
   public transient ExpectedException thrown = ExpectedException.none();
 
   /**
+   * A {@link SimpleFunction} to test that the coder registry can propagate coders
+   * that are bound to type variables.
+   */
+  private static class PolymorphicSimpleFunction<T> extends SimpleFunction<T, T> {
```

Comment:
Maybe a little odd that here `PolymorphicSimpleFunction = SimpleFunction<T, T>` but above `PolymorphicSimpleFunction = SimpleFunction<T, Iterable<T>>`. Looks fine either way.